pypasta/Config.py

```diff
@@ -151,10 +151,7 @@ def path(name):
         # analysis results
         self.d_false_positives = path('FALSE_POSTITIVES')
 
-        if self._mode == Config.Mode.MBOX:
-            cluster_dst = 'MBOX_RESULT'
-        else:
-            cluster_dst = 'PASTA_RESULT'
+        cluster_dst = 'PATCH_GROUPS'
```

Comment:
One last thing: You can use 'PATCH_GROUPS' directly below. There's no need for outsourcing it in a variable.